tests/Feature/FishRecordControllerTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Models\User;
+
+class FishRecordControllerTest extends TestCase
+{
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_create()
+    {
+        $response = $this->actingAs(User::find(3))->post(
+            'fish_records',
+            ['fishing_date' => '2020-04-29', 'harbor' => '東京湾', 'fish_name' => 'シーバス', 'size' => 50]
+        );
+
+        $response->assertRedirect(action('FishRecordController@index'));
+    }
+
+    public function test_index()
+    {
+        $response = $this->actingAs(User::find(3))->get('fish_records');
+
+        $response->assertStatus(200)
+            ->assertViewIs('fish_record_list')
+            ->assertSee('釣り記録一覧');
+    }
+
+    public function test_edit()
+    {
+        $response = $this->actingAs(User::find(3))->post('fish_records/edit/5');
+
+        $response->assertStatus(200)
+            ->assertViewIs('fish_record_edit')
+            ->assertSee('釣り記録修正');
+    }
+
+    public function test_update()
+    {
+        $response = $this->actingAs(User::find(3))->post(
+            'fish_records/update/5',
+            ['fishing_date' => '2020-04-29', 'harbor' => '東京湾', 'fish_name' => 'シーバス', 'size' => 50]
+        );
+
+        $response->assertRedirect(route('list'));
+    }
+
+
+
+    public function test_detalis()
```

Comment:
他のところもtypoしているので、気が向いたら一緒に直しちゃいましょう！
```suggestion
    public function test_details()
```